Lib/concurrent/futures/process.py

```diff
@@ -87,7 +87,12 @@ def close(self):
         self._reader.close()
 
     def wakeup(self):
-        self._writer.send_bytes(b"")
+        try:
```

Comment:
Maybe a _closed flag might be better than catching the exception? You could also then check the flag in the clear method?

Reply:
Yes you are right, this looks better.